qt/python/mantidqt/utils/asynchronous.py

```diff
@@ -16,12 +16,20 @@
 import time
 from traceback import extract_tb
 
+from mantid.py3compat.enum import Enum
+
+
+class TaskExitCode(Enum):
+    OK = 0
```

Comment:
Does python have a list of what it returns when `python script.py` runs? The ["official" documentation](https://docs.python.org/3.6/library/sys.html#sys.exit) says what you have, but I didn't know if there was more guidance elsewhere.

Reply:
> Does python have a list of what it returns when python script.py runs?

Do you mean the exit codes? I've just gone by the Unix ones, 0 OK, 1 ERR, and a custom 2 for a syntax error. As this is a process exit code it's probably OS related rather than Python related

Reply:
That's all that I have found either. I think what you did is fine.